libraries/Matter/src/MatterEndpoints/MatterDimmableLight.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
Please add the Espressif's license header (IDK if the other Matter files might be missing it)

Reply:
I'm do that latter for all the files.

Reply:
done to all source code files.